src/itest/kotlin/nl/info/zac/itest/config/ProjectConfig.kt

```diff
@@ -65,6 +67,7 @@ class ProjectConfig : AbstractProjectConfig() {
         // override default entrypoint for ZAC Docker container to add JaCoCo agent for recording integration test coverage
         "ZAC_DOCKER_ENTRYPOINT" to
             "java" +
+            "-agentlib:jdwp=transport=dt_socket,server=y,suspend=y,address=*:5005" +
```

Comment:
Is this part of the PR or left from a debugging session?

Reply:
This is indeed a debugging entry, but I think it's ok to keep, with a minor change:
```suggestion
            "-agentlib:jdwp=transport=dt_socket,server=y,suspend=n,address=*:5005" +
```

Reply:
This way you can attach a debugger to to running container application. With this debug port made available in the dockerfile and mapped in the dockercompose.

Reply:
This will slow down the integration tests. Plus we already have debugging capabilities when tests are launched in debug mode via IntelliJ's right click on the directory

Reply:
Yes, exactly. I'm afraid this will slow down our itest. I would not do this. If want to debug ZAC while running itests that's fine but just make a code comment out of it since most people won't need it. And especially GitHub won't. ;-)

Reply:
Why did you resolve this conversation @nischitha-2024 without changing the code or making a case why this change should be in here? I still think it should not be added to the code because I think it will slow down our integration tests. Or does it not?